### PR TITLE
Add apple_support 0.13.0

### PR DESCRIPTION
**Pull request by keith**

### CODE REVIEW

File: modules/apple_support/0.12.1/MODULE.bazel [hunk 1]
```diff
@@ -0,0 +1,9 @@
+module(
+    name = "apple_support",
+    version = "0.12.1",
+    compatibility_level = 1,
+)
+
+bazel_dep(name = "bazel_skylib", version = "1.1.1")
+bazel_dep(name = "platforms", version = "0.0.4")
+bazel_dep(name = "stardoc", version = "0.5.0", repo_name = "io_bazel_stardoc", dev_dependency = True)
```
Comment (meteorcloudy) [inline modules/apple_support/0.12.1/MODULE.bazel:9]: Note that, there is a known issue with stardoc and Bzlmod: https://github.com/bazelbuild/bazel/issues/14140, but it's fine for now as a dev dependency.

File: modules/apple_support/0.12.1/source.json [hunk 2]
```diff
@@ -0,0 +1,4 @@
+{
+    "integrity": "sha256-xgS3WGXAf0WCjH//1f2/+BQVqehKaPcanB2OOyaU5Uc=",
```
Comment (meteorcloudy) [inline modules/apple_support/0.12.1/source.json:2]: I calculated `sha256-9t8AfuoSKBgBrWym0XBzu9/pXSm1c4u9byVGYRyDOxw=`

**Review by meteorcloudy — CHANGES_REQUESTED** [review]

File: modules/apple_support/0.12.1/MODULE.bazel (hunk 1 above)
Comment (keith) [reply modules/apple_support/0.12.1/MODULE.bazel:9]: Ah yea, I also filed https://github.com/bazelbuild/stardoc/issues/117

File: modules/apple_support/0.12.1/source.json (hunk 2 above)
Comment (keith) [reply modules/apple_support/0.12.1/source.json:2]: Where are the instructions on how to calculate this? I nabbed the code from the python to do it

Comment (meteorcloudy) [reply modules/apple_support/0.12.1/source.json:2]: See https://developer.mozilla.org/en-US/docs/Web/Security/Subresource_Integrity
Hmm, the code from python should do it correctly, not sure what went wrong.

Comment (keith) [reply modules/apple_support/0.12.1/source.json:2]: What files specifically are you running this on?

```
% cat apple_support.0.12.1.tar.gz | openssl dgst -sha256 -binary | openssl base64 -A
xgS3WGXAf0WCjH//1f2/+BQVqehKaPcanB2OOyaU5Uc=
```

This usage matches what I had here

Comment (meteorcloudy) [reply modules/apple_support/0.12.1/source.json:2]: Oh, I see, the url is wrong, it should be 0.12.1

Comment (keith) [reply modules/apple_support/0.12.1/source.json:2]: Sorry about that! I had to bump mid way through

**Comment by meteorcloudy** [comment]
https://github.com/bazelbuild/apple_support/blob/78aa2a3b67569dc92745b8ca8a668183d97b627c/rules/apple_genrule.bzl#L18

Another unnecessary usage of "@build_bazel_apple_support"

**Comment by meteorcloudy** [comment]
@keith Can you fix the "@build_bazel_apple_support" usage in this version?

**Comment by keith** [comment]
Updated to a new version with that fix

**Review by meteorcloudy — APPROVED** [review]
Perfect, thank you!

**Pull request merged**